app/controllers/api/schemes_controller.rb

```diff
@@ -20,7 +20,7 @@ def show
     end
 
     def index
-
+      @schemes = Scheme.all
```

Comment:
will change after I sort out user sessions in development (current_user was nil)